src/garage/envs/gym_env.py

```diff
@@ -212,6 +216,17 @@ def step(self, action):
 
         observation, reward, done, info = self._env.step(action)
 
+        # check that env_infos are consistent
+        if not self._env_info:
+            self._env_info = {k: type(info[k]) for k in info}
+        elif self._env_info.keys() != info.keys():
+            raise RuntimeError('GymEnv outputs inconsistent env_info keys.')
+        else:
+            info_types = {k: type(info[k]) for k in info}
+            if self._env_info != info_types:
+                raise RuntimeError(
+                    'GymEnv outputs inconsistent env_info shapes.')
```

Comment:
This should say "inconsistent env_info types", I think. I'm not actually sure if we should have this check at all though, since outputting `None` some epochs might be an okay thing to do.